src/Requestor.php

```diff
@@ -37,6 +41,16 @@ public function __construct($apiKey, $apiUrl, $useSsl = true)
         $this->useSsl = $useSsl;
     }
 
+    /**
+     * Set the data to be used as query parameters for this requestor
+     *
+     * @param array $params The data for the query parameters
```

Comment:
Could you update this langugae to "A set of key/value pairs representing each query argument and its value

Reply:
Updated